inc/settings.php

```diff
@@ -84,8 +84,7 @@ function wpcs_options_page() {
 		'WP Conference Schedule',
 		'manage_options',
 		'wp-conference-schedule',
-		'wpcs_options_page_html',
-		'dashicons-schedule'
```

Comment:
Remove to prevent notice of doing wrong as this parameter is for position, not icon.